contrib/packaging/rpm/skydive.spec

```diff
@@ -30,6 +37,9 @@ Summary:        Real-time network topology and protocols analyzer.
 License:        ASL 2.0
 URL:            https://%{import_path}
 Source0:        https://%{import_path}/releases/download/v%{source}/skydive-%{source}.tar.gz
+Source1:        skydive.pp
+Source2:        skydive.if
+Source3:        skydive_selinux.8
```

Comment:
nit: it seems the convention is "-" instead of "_"

Reply:
selinux scripts generate the files like that, but yeah we can change it